userguide/system.rst

```diff
@@ -2288,21 +2288,52 @@ Updating an HA System
 
 If the %brand% array has been configured for High Availability
 (HA), the update process must be started on the active |ctrlr-term|.
-Once the update is complete, the standby |ctrlr-term| will automatically
-reboot. Wait for it to come back up by monitoring the remote console or
-the |web-ui| of the standby |ctrlr-term|.
+The update process is to download and synchronize the update files
+between both |ctrlrs-term|, update and reboot the standby |ctrlr-term|,
+manually activate the standby |ctrlr-term|, then install the update on
```

Comment:
```suggestion
manually activate the standby |ctrlr-term| after the update has completed successfully, then install the update on
```

Reply:
This is already communicated by the previous phrase about updating and rebooting the standby |ctrlr-term|. To put another phrase here about after successful update is both redundant and confuses the order of events.